src/sentry/api/helpers/span_analysis.py

```diff
@@ -0,0 +1,74 @@
+# example data = list of dictionaries
+
+# data = [{'period': 1, 'k': 'browser', 'txn_count': 2978, 'sum_v': 2088325.678, 'count_v': 24069, 'avg_v': 86.76412306},
+# {'period': 1, 'k': 'http.client', 'txn_count': 2978, 'sum_v': 13444332.97, 'count_v': 78114, 'avg_v': 172.1116953},
+# {'period': 1, 'k': 'mark', 'txn_count': 2978, 'sum_v': 0, 'count_v': 5390, 'avg_v': 0},
+# {'period': 1, 'k': 'pageload', 'txn_count': 2978, 'sum_v': 1853836.58, 'count_v': 2978, 'avg_v': 622.5106045},
+# {'period': 1, 'k': 'paint', 'txn_count': 2978, 'sum_v': 0, 'count_v': 5151, 'avg_v': 0},
+# {'period': 1, 'k': 'resource.css', 'txn_count': 2978, 'sum_v': 449854.7821, 'count_v': 8926, 'avg_v': 50.39825029}]
+
+import collections
+
+
+def span_analysis(data):
+    columns = list(data[0].keys())
+
+    # add in two extra columns
+    count_col = [row["count_v"] for row in data]
+    txn_count = [row["txn_count"] for row in data]
+    avg_col = [row["avg_v"] for row in data]
+    sum_col = [row["sum_v"] for row in data]
+
+    relative_freq = [count_col[x] / txn_count[x] for x in range(len(count_col))]
```

Comment:
Talked about this before, I'm going to remove the average duration calculation from my PR and we're going to put it here.